wurst/systems/entities/bosses/AlligatorMan.wurst

```diff
@@ -2,7 +2,30 @@ package AlligatorMan
 import ID
 import Hostile
 import LinkedList
+import BossUtilities
+import RegisterEvents
+import HashMap
 
+//Regions for the alligator islands
+public let NW_ALLIGATOR_REGION = CreateRegion()
```

Comment:
You don't reference these further than creating the overall list, so I would just define them inline there to simply it.

    let regions = asList(
        CreateRegion..addRect(gg_rct_NW_ALLIGATOR),
        ...
    )

Reply:
I'm on the fence about whether this is a constant with regards to using uppercase for it. You may want to modify it later.  I typically only use uppercase for simple types that can't be modified at all, like integers and strings.